packages/text-input-react/src/TextInput.tsx

```diff
@@ -64,7 +67,16 @@ export const TextInput = forwardRef<HTMLInputElement, Props>((props, ref) => {
                 density={density}
                 htmlFor={uid}
             >
-                {label}
+                {!tooltipProps && label}
+                {tooltipProps && (
+                    <>
+                        {label.substring(0, label.lastIndexOf(" "))}
+                        <span style={{ whiteSpace: "nowrap" }}>
+                            {`${label.substring(label.lastIndexOf(" "))} `}
+                            <Tooltip {...tooltipProps} />
+                        </span>
```

Comment:
![](https://media2.giphy.com/media/26AHJpsEO2WZImTpC/giphy.gif?cid=ecf05e47tuvxwc8ku0vygmzmu0bp57rhsr99g17ccr1olr1o&rid=giphy.gif&ct=g)

Reply:
Nå vokser ikke boksen med innholdet, jeg vet ikke hva som er tenkt her? :) Hva med å sette `min-width: fit-content`?

Reply:
Problemet jeg prøver å løse er tooltip havner på en egen linje, som i skjermbildet fra #3223. Pio fikset så den beholder formen, men den kan fortsatt havne på en linje for seg selv.

Reply:
Denne var jo interessant 🤔 Ville forventet ut fra koden at det siste ordet og Tooltip skulle stått alene på egen linje, men kanskje CSS virkelig ikke liker [orphans](https://fonts.google.com/knowledge/glossary/widows_orphans)?

Er vel kanskje uansett det nærmeste man kommer i å prøve å fremtvinge oppførselen vi ønsker oss 😅 Bra jobba, selv om jeg også grøsser litt av inline styling-hacks.

https://user-images.githubusercontent.com/25739615/201683137-33575e19-ea2f-4966-8269-92dc0d04ec94.mov